chipy_org/apps/meetings/views.py

```diff
@@ -200,47 +199,45 @@ def get_context_data(self, **kwargs):
 class RSVPlistCSVBase(RSVPlist):
 
     def _lookup_rsvps(self, rsvp):
+
+        # yield the CVS header
         if self.private:
             yield ["User Id", "Username", "Full Name",
                    "First Name", "Last Name", "Email", "Guests", ]
         else:
             yield ["Full Name", "First Name", "Last Name", "Guests", ]
+
+        data = []
```

Comment:
I think we should switch to a different queryset rather than building an in-memory array here.

Reply:
@raymonndberg as we discussed, a better solution would be completely populating the RSVP model from anonymous user input or the logged in user model with a forced user review of their information.   Either way, the user should be able to review and update the RSVP to correct their information or modify their status.   Also I think we should make email mandatory as somewhat reasonable unique identifier.